tests/MailsMockTraitTest.php

```diff
@@ -118,6 +118,20 @@ public function testMailWithExport()
         $this->assertFileExists($this->getFixturePath('test_mail_with_export.html'));
     }
 
+    public function testMailWithGlobalExportMode()
+    {
+        putenv('FAIL_EXPORT_JSON=false');
+        $this->globalExportMode = true;
+
```

Comment:
let's remove `test_mail_with_global_export.html ` in the start of test case to be sure that artefacts from the previous run is not affect for the current run